perf: Added CAPATH to the cmd parameter

ZacharyBear opened the pull request:
Then developers can use the `make start` to start the **resp-api** application.

ZacharyBear committed:
perf: Added CAPATH to the cmd parameter

ZacharyBear committed:
fix(swaggo): Now allow IPAddresses field to be accept from APIs

ZenkieBear commented:
In the second commit `fix(swaggo): Now allow IPAddresses field to be accept from APIs`, issue #40 is fixed.

ZacharyBear committed:
fix: Modified PEM file type string

The standard PEM key file should named "RSA PRIVATE KEY" or "RSA PUBLIC KEY".

fixed: #42
